src/js/components/message.jsx

```diff
@@ -73,15 +70,9 @@ export class MessageComponent extends Component {
             <img className={ avatarClass.join(' ') }
                  src={ avatarUrl } />;
 
-        const extraSpacingStyle = {
-            marginBottom: '5px',
-            display: 'inline-block'
-        };
-
         const textPart = hasText && <TextMessage {...this.props}
-                                                 style={ hasActions || hasImage ? extraSpacingStyle : null } />;
-        const imagePart = hasImage && <ImageMessage {...this.props}
-                                                    style={ hasActions ? extraSpacingStyle : null } />;
+                                                 className={ `sk-message-item sk-message-text${lastItem === 'text' ? ' sk-last-item' : ''}` } />;
```

Comment:
I'd prefer to build this class list in an array and `.join` instead of doing it on a long line like that